### ISSUE TITLE
Availability of hostmetrics receiver in was Otel

### ISSUE THREAD

**Issue by saaish** [aws-observability/aws-otel-collector#556]
It would be great to have host metrics receiver enabled in aws Otel which will let us OS metrics and give the ability to use a single tool for collecting metrics

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mxiamxia** [comment]
Thanks @saaish, we'll add this request in ADOT roadmap. @jhnlsn @alolita

**Comment by alolita** [comment]
Hi @Konig-Corey we're looking to prioritize this request following metrics stability on OTEL.

**Comment by georges-git** [comment]
It is very important to have hostmetrics sent via OTEL as some the those are used for SLO/SLA managament. Check-https://github.com/aws-observability/aws-otel-collector/issues/1098

**Comment by alolita** [comment]
We plan to add this in an upcoming release @georges-git

**Comment by alexlokshin-czi** [comment]
Is this feature still on the roadmap?

**Comment by mhausenblas** [comment]
ADOT PM here. Yes @alexlokshin-czi it is on our mid-term roadmap. Can you please share your use case and with that help us prioritize it?

**Comment by alexlokshin-czi** [comment]
@mhausenblas , we're replacing prometheus exporters with ADOT/OTEL collectors, and ran into one limitation - host level metrics are not exported with the ADOT collector. Basically, host level metrics are needed for platform monitoring, without them ADOT is not production ready for us.

**Comment by alexlokshin-czi** [comment]
> Thank you @alexlokshin-czi and just to confirm: are you currently using the Prometheus Node exporter for that?

Yes, we're currently using the prometheus node exporter. Our goal is to be able to use ADOT for logs, metrics, and traces -- without having to mix and match agents.

**Comment by igordust** [comment]
Hostmetrics could be very useful also in the scenarios where we want to monitor plain EC2 instances, this could allow us to have a single agent and funnel all signals (logs, metrics and traces) with a single tool.

**Comment by mhemken-vts** [comment]
Hostmetrics is required for us to use otelcol to send metrics to datadog. Would be nice if we could use ADOT out of the box for this use case.

**Comment by danmrichards** [comment]
Moving conversation here given my [PR ](https://github.com/aws-observability/aws-otel-collector/pull/2906) was closed by @mhausenblas. Thank you for the clarification on the contributing guidelines.

Do you have an update on when/if `hostmetrics` will be available in the ADOT collector? You [mentioned last year](You https://github.com/aws-observability/aws-otel-collector/issues/556#issuecomment-1432682520) that it was on the mid-term roadmap.

**Comment by mhausenblas** [comment]
Thanks for your understanding @danmrichards and we don't have an ETA for it yet. I will update this issue as soon as we do.

**Comment by jstephens986** [comment]
Would love to see this part of ADOT.

**Comment by nbcchen** [comment]
> Hostmetrics is required for us to use otelcol to send metrics to datadog. Would be nice if we could use ADOT out of the box for this use case.

Are you using ECS? You can deploy a Datadog Agent for metrics

https://docs.datadoghq.com/containers/amazon_ecs/?tab=awscli#setup
